js/lib/beautify-html.js

```diff
@@ -536,6 +545,31 @@
                 return comment;
             };
 
+            this.get_handlebars_comment = function(start_pos) { //function to return handlebars comment content in its entirety
```

Comment:
Any reason why this wouldn't be part of the general comment method above? it's mostly the same...